configs/candidates/INCLUDE

```diff
@@ -0,0 +1 @@
+base
```

Comment:
I think you inverted updates and candidates.

Expected:
* `testing` requires `candidates`
* `candidates` requires `updates`
* `updates` requires `base`

Reply:
fixed